### PR TITLE
docs: add FAQ about paths to being a moderator

### LINKED ISSUE #44408: FAQ about community leadership
We want to encourage offering help as a leader within the community and document something in the FAQ of our contributing docs. 

The idea is to point interested people in the right direction, whereby they can offer meaningful help and establish their credibility over a period of time. This should essentially help them in becoming a member of our awesome @freeCodeCamp/moderators team.

Please see more context in https://github.com/freeCodeCamp/freeCodeCamp/pull/44399#discussion_r764152816

### PR DESCRIPTION
**Pull request by erictleung**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #44408

These suggestions are probably not exhaustive, nor entirely correct (because I'm not as versed in the other sub-communities, e.g., the forums). Please suggest any changes that you feel might be helpful.

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -69,6 +69,43 @@ We typically do not assign issues to anyone other than long-time contributors. I
    - Did you follow the pull request checklist?
    - Did you give your pull request a meaningful title?
 
+### I am interested in being a moderator at freeCodeCamp. Where should I start?
+
+Our community moderators are our heros. Their voluntary contributions make
+freeCodeCamp make freeCodeCamp a safe and welcoming community.
```
Comment (nhcarrigan) [inline docs/FAQ.md:75]:
```suggestion
Our community moderators are our heroes. Their voluntary contributions make
freeCodeCamp a safe and welcoming community.
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
This is awesome. Thanks for getting this going! One tiny fix, and then I think this is good to go.

> These suggestions are probably not exhaustive, nor entirely correct

It may not be comprehensive, but it is definitely a great starting point that we can iterate on as needed. 🙂

**Comment by erictleung** [comment]
@nhcarrigan thanks for taking a look! Nice spelling catch. In looking at the plural of hero, TIL that Heros is a kind of fish in South America https://en.wikipedia.org/wiki/Heros_(fish)

**Review by naomi-lgbt — APPROVED** [review]
🐟

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
Eric, Thanks for taking the time to contribute. This is going to be of help quite a lot.

**Pull request merged**